### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by deniseyu**
Signed-off-by: Denise Yu <dyu@pivotal.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ I got the basic idea working, even if it doesn't look particularly pretty:
 <img src="/Screenshot_20171116-230003.png" width="250" />
 
 The major problem I ran into was the Deutsche Bahn API itself. 90% of the time, the server responds with 500 errors, making an app like this unusable. To get around this for the hackathon, I manually downloaded some data from their API and saved it as sample static data here. By default, the app uses this very incomplete static data instead of querying the Deutsche Bahn API. (This is why there are so many "UNKNOWN" destinations in the screenshots.) If you decide to take this code and use it against the real API, note that you'll need to provide your API access token in a string resource named "db_api_key". Hopefully with time, their API will improve and an app like this will be possible.
+
+Testing asdfsklfdasfksajjasdf
```
Comment (jenspinney) [inline README.md]: Hm. Not sure about this.

**Review by jenspinney — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jenspinney) [reply README.md]: Hmm, still considering...

Comment (deniseyu) [reply README.md]: Everything is fixed!!!!!!

**Comment by jenspinney** [comment]
Not up to snuff.

**Pull request closed without merging**